src/web_app_skeleton/script/setup.ecr

```diff
@@ -22,7 +22,7 @@ print_done
 <%- end -%>
 
 notice "Installing shards"
-shards install | indent
+shards install --ignore-crystal-version | indent
```

Comment:
Is this a "hey i need this to work for now" contribution which should be removed? Or should this be kept in?

Reply:
This was a tough one... Without it, all tests again < 1.2.2 fail because of https://github.com/luckyframework/lucky_sec_tester/blob/346ceb3567110e433863dd1997afcec6243db3e5/shard.yml#L7 That's an easy one to drop down, but then we can only drop down to 1.1.1 https://github.com/NeuraLegion/sec_tester/blob/9db160150b4cc15ebb6d6722a25ee55a5c5af3ce/shard.yml#L7 and CLI tests against 1.0.0 https://github.com/luckyframework/lucky_cli/blob/89be92657d24128cd2623198e62c13bed1a3ee1b/.github/workflows/ci.yml#L35

Do I just raise the min test to 1.1.1 for CLI since 1.4 comes out tomorrow? Then I could update the other shard, and remove this

Reply:
Alright, I updated the LuckySecTester to be >=1.1.1. It would still fail due to checking 1.0.0. So I still have to either use the flag, or drop 1.0.0 support :thinking:  I'm sort of ok with going with the latter. I feel like once 1.4.0 drops, few people will be wanting to upgrade Lucky and continue using 1.0.0....

Reply:
I don't have any reason _not_ to have this flag in there, I was just curious. It's complicated, and I get that.